README.md

```diff
@@ -533,6 +571,63 @@ project.getMetadata(function(err, metadata) {
 ```
 
 
+## Google Cloud Speech (Limited Preview)
+
+> **This is a Limited Preview release of Google Cloud Speech.** This feature is not covered by any SLA or deprecation policy and may be subject to backward-incompatible changes.
+
+- [API Documentation][gcloud-speech-docs]
+- [Official Documentation][cloud-speech-docs]
+
+```js
+var gcloud = require('gcloud');
+
+// Authorizing on a per-API-basis. You don't need to do this if you auth on a
+// global basis (see Authorization section above).
+
+var speech = gcloud.speech({
+  projectId: 'my-project',
+  keyFilename: '/path/to/keyfile.json'
+});
+
+// Detect the speech in an audio file.
+speech.detect('./audio.raw', {
+  encoding: 'LINEAR16',
+  sampleRate: 16000
+}, function(err, result) {
+  // result = {
+  //   transcript: 'how old is the Brooklyn Bridge',
+  //   confidence: 0.97234234234
+  // ]
+});
+```
+
+```js
+// Detect the speech in an audio file stream.
+var initialRequest = {
+  encoding: 'LINEAR16',
+  sampleRate: 16000,
+  continuous: true
+};
+
+var stream = speech.recognize(initialRequest);
+  .on('data', function (recognizeResponse) {
+    console.log(recognizeResponse);
+  });
+
+stream.write({
+  audioRequest: {
```

Comment:
Just kind of a general thought in regards to the async/sync naming. In node it's pretty common to assume all methods are async by default and for any synchronous version to have the same name with `Sync` appended to it (e.g. `readFile` and `readFileSync`). 

I'm not sure if this convention fits here since these methods are not synchronous on the client side, but I figured I'd throw this thought out there and see how everyone felt about it.